ignite/src/main/java/org/hibernate/ogm/datastore/ignite/impl/IgniteCacheInitializer.java

```diff
@@ -185,9 +199,9 @@ private Class getEntityIdClassName( String table, SchemaDefinitionContext contex
 		CacheConfiguration<?,?> cacheConfiguration = new CacheConfiguration<>();
 		cacheConfiguration.setStoreKeepBinary( true );
 		cacheConfiguration.setSqlSchema( QueryUtils.DFLT_SCHEMA );
-		cacheConfiguration.setBackups( 1 );
 		cacheConfiguration.setName( StringHelper.stringBeforePoint( entityKeyMetadata.getTable() ) );
 		cacheConfiguration.setAtomicityMode( CacheAtomicityMode.TRANSACTIONAL );
+		cacheConfiguration.setCacheMode( CacheMode.PARTITIONED );
```

Comment:
it is unacceptable! 
Why you configure default CacheMode ?

Reply:
Why not? PARTITIONED is the default mode. Do you think that REPLICATED is better for entity cache by default?

Reply:
Hi @Z-z-z-z !

I think property file is  the best way for configure cache mode of each cache. Your currect code blocks using other cache modes. And this is unacceptable. In this way, please, could you remove  cache mode configuration from code to property file?
Thanks!